doc/api/esm.md

```diff
@@ -1456,6 +1456,10 @@ _internal_, _conditions_)
 
 > Stability: 1 - Experimental
 
+> **Note: Do not rely on this flag. We plan to remove it once the
```

Comment:
I think I made the same suggestion (to use passive voice) months ago to @Trott and he pointed me to the style guide: https://docs.microsoft.com/en-us/style-guide/grammar/person. Basically, “we” is discouraged, but it’s still better than the passive voice. Whoever is the authority on style should tell me how this should be phrased.

Reply:
In general, the Microsoft Style Guide prefers active voice, but suggests passive voice for some situations. In particular, it suggests that warnings and notifications might be better as passive voice. I would consider this a warning/notification, so passive voice is likely appropriate.

Ref: https://docs.microsoft.com/en-us/style-guide/grammar/verbs#active-and-passive-voice

Reply:
I feel like I grew up being taught that technical documentation should be in the passive voice, but clearly the modern style is active. If anyone has an idea for a replacement for “we” I would consider that, but I think “we” is better than the passive voice.

Reply:
> I feel like I grew up being taught that technical documentation should be in the passive voice, but clearly the modern style is active.

Yeah, pretty much. The passive voice makes things sound formal, so published documentation adopted that for a long time. At some point, people decided that documentation (especially on the web) should be friendly and relatable rather than formal and authoritative-sounding, and active voice was adopted. Additionally, passive voice is supposedly harder to understand for a lot of folks whose first language is not English. But neither passive voice nor active voice are "wrong".

> If anyone has an idea for a replacement for “we” I would consider that, but I think “we” is better than the passive voice.

```markdown
> **Note: Do not rely on this flag. It will get the boot once the
```